packages/jmespath/src/PowertoolsFunctions.ts

```diff
@@ -10,7 +12,9 @@ const decoder = new TextDecoder('utf-8');
  *
  * Built-in JMESPath functions include: `powertools_json`, `powertools_base64`, `powertools_base64_gzip`
  *
- * You can use these functions to decode and/or deserialize JSON objects when using the {@link index.search | search} function.
+ * You can use these functions to decode and/or deserialize JSON objects when using the {@link search | `search`} function.
+ *
+ * When using the {@link extractDataFromEnvelope | extractDataFromEnvelope} uses these functions internally.
```

Comment:
This sentence reads a bit off to me. What's it trying to say?

Reply:
It's trying to say that when you're using `extractDataFromEnvelope` the function being documented is used internally - which I guess is useless information hah.

I'm gonna remove it.